server/routes/api/documents.ts

```diff
@@ -37,150 +38,176 @@ import {
   presentDocument,
   presentPolicies,
 } from "@server/presenters";
-import slugify from "@server/utils/slugify";
 import {
-  assertUuid,
-  assertSort,
-  assertIn,
-  assertPresent,
-  assertPositiveInteger,
-  assertNotEmpty,
-  assertBoolean,
-} from "@server/validation";
+  DocumentsListReqSchema,
+  DocumentsListReq,
+  DocumentsArchivedReqSchema,
+  DocumentsArchivedReq,
+  DocumentsDeletedReqSchema,
+  DocumentsDeletedReq,
+  DocumentsViewedReqSchema,
+  DocumentsViewedReq,
+  DocumentsDraftsReqSchema,
+  DocumentsDraftsReq,
+  DocumentsInfoReqSchema,
+  DocumentsInfoReq,
+  DocumentsExportReqSchema,
+  DocumentsExportReq,
+  DocumentsRestoreReqSchema,
+  DocumentsRestoreReq,
+  DocumentsSearchTitlesReqSchema,
+  DocumentsSearchTitlesReq,
+  DocumentsSearchReqSchema,
+  DocumentsSearchReq,
+  DocumentsTemplatizeReqSchema,
+  DocumentsTemplatizeReq,
+  DocumentsUpdateReqSchema,
+  DocumentsUpdateReq,
+  DocumentsMoveReqSchema,
+  DocumentsMoveReq,
+  DocumentsArchiveReqSchema,
+  DocumentsArchiveReq,
+  DocumentsDeleteReqSchema,
+  DocumentsDeleteReq,
+  DocumentsUnpublishReqSchema,
+  DocumentsUnpublishReq,
+  DocumentsImportReqSchema,
+  DocumentsImportReq,
+  DocumentsCreateReqSchema,
+  DocumentsCreateReq,
+} from "@server/routes/api/types";
```

Comment:
I think this is a case where it's better to just do something like...

```
import * as T from "@server/routes/api/types";

T.DocumentsListSchema
```

Maybe we can lose `Req` as they all have it?

Reply:
> Maybe we can lose Req as they all have it?

Didn't quite get. You mean rename, for example `DocumentsCreateReqSchema` -> `DocumentsCreateSchema` but keep `DocumentsCreateReq` unchanged?

Reply:
I have `Req` there purposely to distinguish with `Res`, in case if we have that in future, something like
```js
APIContext<DocumentsCreateReq, DocumentsCreateRes>
```

Reply:
ok

Reply:
> rename, for example `DocumentsCreateReqSchema` -> `DocumentsCreateSchema` but keep DocumentsCreateReq unchanged?

Sorry, to be clear – this sounds good.